src/MusicStore/Program.cs

```diff
@@ -59,33 +73,49 @@ public static void Main(string[] args)
                 
                 var minRequestTime = long.MaxValue;
                 var maxRequestTime = long.MinValue;
-                var averageRequestTime = 0.0;
+                int N = 1001;
+                long[] responseTimes = new long[N];
 
-                Console.WriteLine("Running 100 requests");
-                for (var i = 1; i <= 100; i++)
+                Console.WriteLine($"Running {N} requests");
+                JitBenchEventSource.Log.BeginRequests(N);
```

Comment:
In the past I think I've observed the warmup period can extend for a few thousand requests before the app really seems to settle into a long term average. You may want to issue a large number of unmeasured requests, or separately measure a pool or warmup requests and a pool of steady-state requests.

Reply:
I'll play with this a bit more -- I can simply wrap this loop in another and either print each batches' data or else just print the last one.